test/integration/JobRoleIntegrationTest.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosResponse } from "axios";
+import { expect } from 'chai';
+import { URL } from "../../src/services/JobRoleService";
+
+describe('JobRoleIntegration', function () {
+    it('getOpenJobRoles should return list of job roles', async () => {
+        try {
+            const response: AxiosResponse = await axios.get(URL);  
+            
+            const data = response.data;
+            const status = response.status;
+
+            expect(status).to.equal(200);
+            expect(data).to.not.be.null;
+        } catch (e) {
+            return;
+        }
+    });
+})
```

Comment:
I guess you'll have the same problem here running the integration test as the frontend? Something to bare in mind

Reply:
Added in a skip for the test and updated docs